tests/std/tests/P0896R4_views_drop/test.cpp

```diff
@@ -526,4 +543,13 @@ int main() {
         STATIC_ASSERT(test_one(v, only_four_ints));
         test_one(v, only_four_ints);
     }
+
+    { // Validate that we can use something that is convertible to integral (#1957)

```

Comment:
Although we believe that GitHub is our forever home, we should explicitly cite GH-1957 to avoid any confusion with previous bug databases. Also occurs in `P0896R4_views_take/test.cpp`.

Outside of repo sources, it's totally fine to cite with `#` - it's just that the repo mentions bugfixes across decades.